definitions/npm/lodash_v4.x.x/flow_v0.63.x-/test_lodash-fp-v4.x.x.js

```diff
@@ -206,118 +209,130 @@ function square(n) {
 }
 
 map(square, [4, 8]);
-map(square, { 'a': 4, 'b': 8 });
+map(square, { a: 4, b: 8 });
 
-var users = [
-  { 'user': 'barney' },
-  { 'user': 'fred' }
-];
+var users = [{ user: "barney" }, { user: "fred" }];
 
 // The `property` iteratee shorthand.
-map('user', users);
-
+map("user", users);
 
 /**
  * pullAllBy
  */
-pullAllBy('x', [{ 'x': 1 }, { 'x': 3 }], [{ 'x': 1 }, { 'x': 2 }, { 'x': 3 }, { 'x': 1 }]);
-pullAllBy('x')([{ 'x': 1 }, { 'x': 3 }])([{ 'x': 1 }, { 'x': 2 }, { 'x': 3 }, { 'x': 1 }]);
-
+pullAllBy("x", [{ x: 1 }, { x: 3 }], [{ x: 1 }, { x: 2 }, { x: 3 }, { x: 1 }]);
+pullAllBy("x")([{ x: 1 }, { x: 3 }])([{ x: 1 }, { x: 2 }, { x: 3 }, { x: 1 }]);
 
 /**
  * unionBy
  */
 unionBy(Math.floor, [2.1], [1.2, 2.3]);
-unionBy('x', [{ 'x': 1 }], [{ 'x': 2 }, { 'x': 1 }]);
-
+unionBy("x", [{ x: 1 }], [{ x: 2 }, { x: 1 }]);
 
 /**
  * uniqBy
  */
 uniqBy(Math.floor, [2.1, 1.2, 2.3]);
-uniqBy('x', [{ 'x': 1 }, { 'x': 2 }, { 'x': 1 }]);
-
+uniqBy("x", [{ x: 1 }, { x: 2 }, { x: 1 }]);
 
 /**
  * clone
  */
-clone({a: 1}).a == 1;
+clone({ a: 1 }).a == 1;
 // $ExpectError property `b`. Property not found in object literal
-clone({a: 1}).b == 1
+clone({ a: 1 }).b == 1;
 // $ExpectError number. This type is incompatible with function type.
-clone({a: 1}).a == 'c';
+clone({ a: 1 }).a == "c";
 
 /**
  * isEqual
  */
-isEqual('a', 'b');
-isEqual({x: 1}, {y: 2});
-isEqual({x: 1})({y: 2});
+isEqual("a", "b");
+isEqual({ x: 1 }, { y: 2 });
+isEqual({ x: 1 })({ y: 2 });
 
 // $ExpectError function type expects no more than 2 arguments
 isEqual(1, 2, 3);
 
-
 /**
  * range
  */
-range(0, 10)[4] == 4
-range(0)(10)[4] == 4
+range(0, 10)[4] == 4;
+range(0)(10)[4] == 4;
 // $ExpectError string. This type is incompatible with number
-range(0, 'a');
+range(0, "a");
 // $ExpectError string cannot be compared to number
-range(0, 10)[4] == 'a';
-
+range(0, 10)[4] == "a";
 
 /**
  * sortedIndexBy
  */
-sortedIndexBy(function(o) { return o.x; }, { 'x': 4 }, [{ 'x': 4 }, { 'x': 5 }]);
-sortedIndexBy('x', { 'x': 4 }, [{ 'x': 4 }, { 'x': 5 }]);
-sortedIndexBy('x')({ 'x': 4 }, [{ 'x': 4 }, { 'x': 5 }]);
-
+sortedIndexBy(
+  function(o) {
+    return o.x;
+  },
+  { x: 4 },
+  [{ x: 4 }, { x: 5 }]
+);
+sortedIndexBy("x", { x: 4 }, [{ x: 4 }, { x: 5 }]);
+sortedIndexBy("x")({ x: 4 }, [{ x: 4 }, { x: 5 }]);
 
 /**
  * sortedLastIndexBy
  */
-sortedLastIndexBy(function(o) { return o.x; }, { 'x': 4 }, [{ 'x': 4 }, { 'x': 5 }]);
-sortedLastIndexBy('x', { 'x': 4 }, [{ 'x': 4 }, { 'x': 5 }]);
+sortedLastIndexBy(
+  function(o) {
+    return o.x;
+  },
+  { x: 4 },
+  [{ x: 4 }, { x: 5 }]
+);
+sortedLastIndexBy("x", { x: 4 }, [{ x: 4 }, { x: 5 }]);
+
+/**
```

Comment:
These tests were added for sortedUniq and sortedUniqBy, everything else is just my prettier formatting which I didn't think to turn off... 😬 @GAntoine

Reply:
And for the lodash-file, I assumed it was due to the EOL. I practically just saved the file to fix that and committed, then made my changes in the next commit.